### ISSUE TITLE
[YouTube Downloader] ...

### ISSUE THREAD

**Issue by gulomet123** [raycast/extensions#11907]
### Extension

https://www.raycast.com/vimtor/youtube-downloader

### Description

when I try to download a video from yt it says encoding failed on every fromat (mp4, mp3, webm) 
(and yes I am trying to download videos from yt)

### Steps To Reproduce

1. paste the link into the extension
2. pick a format to download from 
3. downloading a video 
4. then getting an error - encoding failed output format mp3/webm/mp4 is not available

### Current Behaviour

Getting an error from the extension that I can't download the video because the format is not available.

### Expected Behaviour

dowloading video

**Comment by vimtor** [comment]
This issue is probably related to #11822. Could you please verify that you have ﻿`ffmpeg` at version 6.1 by running ﻿`ffmpeg -version` in your terminal?

**Comment by gulomet123** [comment]
![Xnip2024-04-20_12-42-20](https://github.com/raycast/extensions/assets/74207143/c2fc2166-0a04-431c-b903-a97e300ee4b1)
I have ffmpeg version 7.0

**Comment by gulomet123** [comment]
I reinstalled the ffmpeg 7 to 6 but the extension doesn't recognize the ffmpeg
![Xnip2024-04-20_15-34-58](https://github.com/raycast/extensions/assets/74207143/32688bb5-db17-41f8-b4be-38431f070ac8)

**Comment by LexDeBash** [comment]
> ![Xnip2024-04-20_12-42-20](https://private-user-images.githubusercontent.com/74207143/324172212-c2fc2166-0a04-431c-b903-a97e300ee4b1.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM2MjI2NjksIm5iZiI6MTcxMzYyMjM2OSwicGF0aCI6Ii83NDIwNzE0My8zMjQxNzIyMTItYzJmYzIxNjYtMGEwNC00MzFjLWI5MDMtYTk3ZTMwMGVlNGIxLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDIwVDE0MTI0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVjY2ZkZWYzYWM2NGVkMmM1MmU3MDAxNTE0ZWUzMDg1OTFhMjNhMTFmMjRhYjVkODg0ZTczZDM1ODkzMzM4M2YmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.dwUlJq2dmZZ8r_vfKkH4DtmxD_Q4si55qLX_VaVSBaY) I have ffmpeg version 7.0

I have the same problem. And I also have version 7.0.

**Comment by LexDeBash** [comment]
I solved the problem as follows: First, I opened the terminal and uninstalled the current version of FFmpeg by running brew uninstall ffmpeg. Then, I installed version 6 of FFmpeg using the command brew install ffmpeg@6. After the installation was complete, I linked this version to ensure it was recognized by the system, using the command brew link ffmpeg@6 --force. Following all these steps, I reinstalled the extension. Now everything is working properly!
